responses/ro/HassClimateGetTemperature.yaml

```diff
@@ -2,5 +2,6 @@ language: ro
 responses:
   intents:
     HassClimateGetTemperature:
-      success:
-        - "{{ state.state }} {{ 'grad' if state.state | abs == 1 else 'grade' if state.state == 0 or state.state % 1 != 0 or state.state % 100 | abs < 20 else 'de grade' }}"
+      default: '{{ state.state }} {{ ''grad'' if state.state | abs == 1 else ''grade''
```

Comment:
I understand the conditional statement formatting but why exactly did we migarate here to single quotes? It just makes the code less readable and does not bring much of a value in situation where we still have hundreds of places using double quotes all over the repository.
Does this have other benefits I'm maybe not aware of?

Reply:
No, this seems to be something PyYAML chose to do during the migration of the response key. We can change it back after the merge; I agree that it's less readable.